src/js/components/Text/stories/TipOnText.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { grommet, Box, Text, Grommet } from 'grommet';
+
+export const TipOnText = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" width="small" pad="small">
+      <Text truncate tip="a b c d e f g h i j k l m n o p q r s t u v w x y z">
```

Comment:
This Text isn't aligned to the center and it looks inconsistent with the below tip examples.
Feel free to align all the examples to start (i.e. remove the align prop since its default to 'start' anyway)